app/views/proposals/index.html.haml

```diff
@@ -6,6 +6,17 @@
         %span.notranslate
           = @conference.title
 
+
+  - if @program.cfp_open?
+    - if @program.cfp.description.length>0
```

Comment:
Why do we need to check the length of the description here?

Reply:
According to me, if organizers are not giving any description we shouldn't make a div that contains "description" and description text

Reply:
I completely agree, we should not create divs we don't actually need. 

But why do we need to check the length of the description and not just if @program.cfp.description

Reply:
I guess, even if organizers leaves the description field empty, description for that particular cfp is a string of zero length and not nil.🤔🤔

Reply:
Then cfp.description.present? which checks against nil and empty string values.

Reply:
That's a nice idea, instead of checking the length😃😃